python/semantic_kernel/connectors/ai/open_ai/services/open_ai_chat_completion_base.py

```diff
@@ -85,6 +88,43 @@ async def complete_chat_with_functions_async(
                 _parse_message(choice.message, self.log) for choice in response.choices
             ]
 
+    async def complete_chat_with_data_async(
```

Comment:
we need to move this new set of functions into the `azure_chat_completion` class since they are azure specific...

Reply:
moved to `azure_chat_completion`